src/Actions/BcgovSettings.php

```diff
@@ -285,6 +318,49 @@ public function bcgov_external_link_icons_settings() {
 		<?php
     }
 
+	/**
+     * Sets the Google Site Name.
+     *
+     * @since 1.2.9
+     * @return void
+     */
+    public function bcgov_mobile_nav_breakpoint_settings() {
+        if ( isset( $_POST['bcgov_mobile_nav_breakpoint_nonce'] ) ) {
+
+			if ( wp_verify_nonce( $nonce, 'bcgov_mobile_nav_breakpoint_nonce' ) ) {
+				if ( isset( $_POST['bcgov_mobile_nav_breakpoint'] ) ) {
+					$mobile_nav_breakpoint = sanitize_text_field( $_POST['bcgov_mobile_nav_breakpoint'] );
+					update_option( 'bcgov_mobile_nav_breakpoint', $mobile_nav_breakpoint );
+				}
+			}
+		}
+
+        // Get the current Google Site Name setting.
+        $mobile_nav_breakpoint = get_option( 'bcgov_mobile_nav_breakpoint_settings', '' );
+
+        if ( is_array( $mobile_nav_breakpoint ) ) {
```

Comment:
is there a chance this will ever be an array

Reply:
Honestly I have to stop "multi-tasking"... sigh / ... No. No it won't. Fixed.